.travis.yml

```diff
@@ -85,7 +85,15 @@ install:
 script:
   - if [ "$PRESTASHOP_TEST_TYPE" = "unit" ]; then
```

Comment:
```suggestion
  - if [ "$PRESTASHOP_TEST_TYPE" = "unit" ]; then
        bash tests/check_file_syntax.sh;
    fi

  - if [ "$PRESTASHOP_TEST_TYPE" = "unit" ]; then
        bash tests-legacy/check_phpunit.sh;
    fi

  - if [ "$PRESTASHOP_TEST_TYPE" = "unit" ]; then
        bash tests/check_unit.sh;
    fi

```

Like below

Reply:
Feels strange to repeat this `if` ... although I know it's to avoid the hardcoded exit

Reply:
Two proper solutions:
 * repeat if statement
 * chain scripts with `&&`

Reply:
Fixed